stdlib/2/__builtin__.pyi

```diff
@@ -30,26 +30,27 @@ class classmethod: pass  # Special, only valid as a decorator.
 class object:
     __doc__ = ...  # type: Optional[str]
     __class__ = ...  # type: type
+    __dict__ = ...  # type: Dict[str, Any]
```

Comment:
This isn't really correct -- `object` itself doesn't have `__dict__` and neither do classes using `__slots__`.